CRM/Civicase/Helper/NewCaseWebform.php

```diff
@@ -8,7 +8,7 @@ class CRM_Civicase_Helper_NewCaseWebform {
   /**
    * Retrieve civicase webform url.
    */
-  public static function appendUrlToOptions(&$options) {
+  public static function addWebformDataToOptions(&$options) {
```

Comment:
Can we change the function description to reflect the name. I.e the function adds web form data to options array.